libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/attributes/AttributeType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing.attributes;
+
+/**
+ * Attribute Type.
+ */
+public enum AttributeType {
```

Comment:
Why do you need this class? The Java's type (String, Long, Double, Boolean) are fully sufficient for us

Reply:
OTEL supports list and map also so may be we need to expand in the future.

Reply:
Also, added to standardize the supported types so that everywhere we shouldn't be doing the instance of check.

Reply:
> OTEL supports list and map also so may be we need to expand in the future.

Still covered by Java types, no need to duplicate what OTEL did

> that everywhere we shouldn't be doing the instance of 

You will do this check only once, if it is everywhere - the implementation is not right